CHANGELOG

```diff
@@ -1,3 +1,5 @@
+2.0.0
+    - Use [selenium-download](https://github.com/groupon/selenium-download) to manage downloading of Selenium server and Chromedriver. This requires node 4+ and Chrome 54 or higher.
```

Comment:
(Nice-to-have) Move the breaking change ("This requires node 4+ and Chrome 54 or higher.") into a separate line with the heading "BREAKING CHANGES", like we do on [Progressive](https://github.com/mobify/progressive-web-sdk/blob/develop/CHANGELOG.md).